composer.json

```diff
@@ -19,15 +19,15 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/framework": "^4.10",
-        "dnadesign/silverstripe-elemental": "^4.0",
-        "silverstripe/userforms": "^5.0",
-        "silverstripe/vendor-plugin": "^1.0"
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "dnadesign/silverstripe-elemental": "5.0.0-alpha2",
```

Comment:
```suggestion
        "dnadesign/silverstripe-elemental": "^5.0.0-alpha2",
```

Reply:
That will allow the installation of later 5.x version.

Reply:
DONE